javascript/exercise2/ex3/index.js

```diff
@@ -0,0 +1,8 @@
+const btn = document.querySelector('.btn');
+const text = document.querySelector('.demoText');
+
+const changeColor = () => {
```

Comment:
Required is to set the background color, not change the color. Please update the naming convention for more clearly